web/scripts/main-index.js

```diff
@@ -127,8 +127,26 @@ function makeDataProvider() {
     };
 }
 
+function generateWeightClasses() {
+    for(var i = 0; i < selClass.options.length; i++) {
+        var option = selClass.options[i];
+        var num = option.text.replace(/[^0-9\.]/g, '');
+        console.log(num);
+        if(option.text.indexOf('lbs') >= 0 && selWeightType.value === "kg") {
+            option.text = option.text.replace(num, Math.round(common.lbs2kg(num)));
```

Comment:
Is this logic a really fancy way to get around the "+" sign? :-)

Instead of doing all this replacing, it would probably be easier to just remember whether there was a "+" sign, and put it back after, always overwriting all the text.